components/clp-py-utils/clp_py_utils/s3_utils.py

```
@@ -304,5 +308,44 @@
+            chunk.append(key_prefix + relative_obj_key)
+            if len(chunk) == chunk_size:
+                yield chunk
+                chunk = list()
+
+        if chunk:
+            yield chunk
+
+    def gen_deletion_config(objects_to_delete: List[str]):
+        return {"Objects": [{"Key": obj_to_delete} for obj_to_delete in objects_to_delete]}
+
+    boto3_config = Config(retries=dict(total_max_attempts=3, mode="adaptive"))
```

Pylint (3.3.7)

[refactor] Consider using '{"total_max_attempts": 3, "mode": 'adaptive'}' instead of a call to 'dict'.

(R1735)